res/skins/Deere (64 Samplers)/skin_settings.xml

```diff
@@ -25,12 +25,12 @@
             <SetVariable name="MaximumSize"><Variable name="SquareButtonMaximumSize"/></SetVariable>
             <SetVariable name="SizePolicy"><Variable name="SquareButtonSizePolicy"/></SetVariable>
             <SetVariable name="state_0_text"></SetVariable>
-            <SetVariable name="state_0_pressed">icon/ic_settings_48px.svg</SetVariable>
-            <SetVariable name="state_0_unpressed">icon/ic_settings_48px.svg</SetVariable>
+            <SetVariable name="state_0_pressed">../Deere/icon/ic_clear_48px.svg</SetVariable>
```

Comment:
Is this actually a unrelated 2.1 bug?

Reply:
you mean the "../Deere " in the path for the icon?

or the change of the icon?

I changed the Icon because I removed the Icon to open the Skin Settings Menu, and therefore It makes more sense to show there an "X" to close the settings menu

Before:
![deere_closed](https://user-images.githubusercontent.com/3403218/39098698-07ba2a4c-466f-11e8-969e-4e447b0aaed1.png)
![deere_open](https://user-images.githubusercontent.com/3403218/39098740-782c9d50-466f-11e8-89e6-d83ba09e6125.png)


with this PR:
![tango_changed_skin_close](https://user-images.githubusercontent.com/3403218/39098724-5e854e60-466f-11e8-8168-948f1cda30d7.png)

Reply:
I like to keep that gear button, but the X for closing is a good Idea.

Reply:
ok